src/Metadata/Storage/TableMetadataStorage.php

```diff
@@ -109,10 +118,20 @@ public function reset(): void
     {
         $this->checkInitialization();
 
+        $tableName = $this->configuration->getTableName();
+
+        if (str_contains($tableName, '.')) {
+            $parts = array_map($this->platform->quoteSingleIdentifier(...), explode('.', $tableName));
```

Comment:
It looks like the meaning of the table name is inconsistent across the project. Here, it's escaped and potentially corrupted (e.g. a value like `migrations."project.v1"`, which is valid for Postgres and SQL Server wouldn't work). In a lot of other cases like this, it would:
 https://github.com/doctrine/migrations/blob/cad2d77234894b5c131416c3b7dd44164b041129/src/Metadata/Storage/TableMetadataStorage.php#L144-L146

I propose to just remove the call to `Connection::quoteIdentifier()` for consistency with the rest of the code.

A proper, longer-term solution would be:
1. Parse the name in the configuration
2. Have `getTableName()` return an `OptionallyQualifiedName`
3. Use `Name::toSQL()` in the code that uses the table name in SQL

Reply:
Quoting removed, the longer term solution can be looked at later.